src/app/create-room/create-room.component.ts

```diff
@@ -0,0 +1,54 @@
+import {Component} from '@angular/core';
+import {
+  FormControl,
+  FormGroup,
+  ReactiveFormsModule,
+  Validators,
+  ɵFormGroupValue,
+  ɵTypedOrUntyped
+} from "@angular/forms";
+import {Router} from "@angular/router";
+
+@Component({
+  selector: 'app-create-room',
+  templateUrl: './create-room.component.html',
+  styleUrls: ['./create-room.component.css']
+})
+export class CreateRoomComponent {
+
+  DEFAULT_DECK: string = "NA; 1h; 1.5h; 2h; 3h; 4h; 6h; 1d; 1.5d; 2d; 3d; 4d; 1w; 1.5w; 2w; 2.5w; 3w";
+
+  checkoutForm: FormGroup;
+
+  constructor(
+    private router: Router
+  ) {
+    this.checkoutForm = new FormGroup<any>({
```

Comment:
I have to add "strictPropertyInitialization": false in tsconfig.json to avoid the error -> Property '...' has no initializer and is not definitely assigned in the constructor
So why is it important?